index.bs

```diff
@@ -3677,7 +3677,7 @@ Content-Type: application/reports+json
   header's \``DENY`\`, and `'self'` to that header's \``SAMEORIGIN`\`. [[!HTML]]
 
   In order to allow backwards-compatible deployment, the
-  <a>`frame-ancestors`</a> directive <em>obsoletes</em> the
+  <a>`frame-ancestors`</a> directive <em>supersedes</em> the
```

Comment:
Actually thinking about it some more, I think this could also be read in the same way as obsoletes. Let's go with overrides:

```suggestion
  <a>`frame-ancestors`</a> directive <em>overrides</em> the
```